### PR TITLE
[PALEMOON] Link dictionaries to addons.palemoon.org

### LINKED ISSUE #933: Unable to install dictionary in palemoon 28.3.0
When I was filling out a webform I wanted to check the spelling in dutch by clicking the right mouse button.
Since I do not have a dutch dictionary installed I chose 'Add Dictionaries'. I was led to [this](https://addons.mozilla.org/en-US/firefox/language-tools/) mozilla(?) page.
I selected the dutch dictionary and was presented with an install button saying '[Only with Firefox-Get Firefox Now](https://addons.mozilla.org/en-US/firefox/addon/woordenboek-nederlands/)'.
It looks like this functionality is broken by mozilla.
Is there another way to add a dictionary?

I am using palemoon in linux mint mate 19.1(64b)

### PR DESCRIPTION
**Pull request by JustOff**
Set `browser.dictionaries.download.url` to `https://addons.palemoon.org/dictionaries/`.

This resolves #933.

Uplift is wanted.

### CODE REVIEW

File: application/palemoon/app/profile/palemoon.js [hunk 1]
```diff
@@ -38,7 +38,8 @@ pref("extensions.strictCompatibility", false);
 pref("extensions.minCompatibleAppVersion", "1.5");
 
 // Preferences for APO integration
-#define APO_AM_URL addons.palemoon.org/integration/addon-manager
+#define APO_URL addons.palemoon.org
+#define APO_AM_URL %APO_URL%/integration/addon-manager
```
Comment (mattatobin) [inline application/palemoon/app/profile/palemoon.js]: Incorrect. Should be @ not %

File: application/palemoon/branding/shared/pref/preferences.inc [hunk 2]
```diff
@@ -1,4 +1,5 @@
-#define APO_AM_URL addons.palemoon.org/integration/addon-manager
+#define APO_URL addons.palemoon.org
+#define APO_AM_URL %APO_URL%/integration/addon-manager
```
Comment (mattatobin) [inline application/palemoon/branding/shared/pref/preferences.inc]: Ditto

File: application/palemoon/app/profile/palemoon.js (hunk 1 above)
Comment (JustOff) [reply application/palemoon/app/profile/palemoon.js]: Oops, sorry, I was sure I checked it out :anguished:

**Comment by mattatobin** [comment]
Would you please revert your branch, do a clean commit and force push?

**Comment by JustOff** [comment]
> Would you please revert your branch, do a clean commit and force push?

@mattatobin, it seems I did something wrong and now it says
```
JustOff:PR_dictionaries_url was force-pushed and no longer has any new commits.
Pushing new commits will allow the pull request to be re-opened.
```
however https://github.com/JustOff/UXP/tree/PR_dictionaries_url **has** new commit ...

**Comment by mattatobin** [comment]
Strange, maybe github busted it.. I've done it before. Oh well, do you want to do a new PR or should I just do it direct?

**Comment by JustOff** [comment]
> Oh well, do you want to do a new PR or should I just do it direct?

Here it is: #975.

**Pull request closed without merging**